projects/view/client/projectProfile/projectProfile.js

```diff
@@ -96,3 +96,104 @@ Template.projectProfile.onCreated(function(){
     self.subscribe('singleProject', projectId);
   });
 });
+
+
+Template.rtProjectProfile.helpers({
+  //Using the Flowrouter package to get the route param projects/:projectId ; is set up in the Template.projectProfile.onCreated callback
+    getProject: function () {
```

Comment:
Rename to `project` instead of `getProject`. This will make code more literate when reading statements such as the following:

* "with project" as opposed to "with get project"
* "if project" as opposed to "if get project"
* "project.title" as opposed to "getProject.title"